src/csharp/Microsoft.Spark.Worker.UnitTest/TaskRunnerTests.cs

```diff
@@ -18,102 +18,100 @@ public class TaskRunnerTests
         [Fact]
         public void TestTaskRunner()
         {
-            using (var serverListener = new DefaultSocketWrapper())
-            {
-                serverListener.Listen();
+            using var serverListener = new DefaultSocketWrapper();
+            serverListener.Listen();
 
-                var port = (serverListener.LocalEndPoint as IPEndPoint).Port;
-                var clientSocket = new DefaultSocketWrapper();
-                clientSocket.Connect(IPAddress.Loopback, port, null);
+            var port = (serverListener.LocalEndPoint as IPEndPoint).Port;
+            var clientSocket = new DefaultSocketWrapper();
+            clientSocket.Connect(IPAddress.Loopback, port, null);
 
-                PayloadWriter payloadWriter = new PayloadWriterFactory().Create();
-                var taskRunner = new TaskRunner(0, clientSocket, false, payloadWriter.Version);
-                var clientTask = Task.Run(() => taskRunner.Run());
+            PayloadWriter payloadWriter = new PayloadWriterFactory().Create();
+            var taskRunner = new TaskRunner(0, clientSocket, false, payloadWriter.Version);
+            var clientTask = Task.Run(() => taskRunner.Run());
 
-                using (ISocketWrapper serverSocket = serverListener.Accept())
-                {
-                    System.IO.Stream inputStream = serverSocket.InputStream;
-                    System.IO.Stream outputStream = serverSocket.OutputStream;
+            using (ISocketWrapper serverSocket = serverListener.Accept())
```

Comment:
There is a scope here (notice that there is an `Assert` after this block. It may be OK, but don't want to explore this in this PR (the other changes are suggested by the IDE).